configure

```diff
@@ -932,6 +937,23 @@ def configure_node(o):
   else:
     o['variables']['node_enable_v8_vtunejit'] = 'false'
 
+  if flavor != 'linux' and (options.enable_lto):
+    raise Exception(
+      'The lto option is supported only on linux.')
+
+  if flavor == 'linux':
+    if options.enable_lto:
+      version_checked = '5.4.1'
+      for compiler in [(CC, 'c'), (CXX, 'c++')]:
+        ok, is_clang, clang_version, compiler_version = \
+          try_check_compiler(compiler[0], compiler[1])
+        if is_clang or compiler_version < version_checked:
```

Comment:
String-based version range checking like this is not foolproof, for example: `'10.0.0' < '5.4.1'` will evaluate to `True`. At the very least we would need a version comparison function that simply splits on '.' and compares each integer value between two version strings.

Reply:
Hello

Thank you for the comment. I will implement this asap.

@octaviansoldea